src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -106,9 +106,12 @@ class CassandraReadJournal(system: ExtendedActorSystem, config: Config)
 
   @volatile private var sessionUsed = false
 
+  private val metricsCategory = s"cassandra.readjournal.${system.name}.hash.${System.identityHashCode(this)}"
```

Comment:
I created an issue for this: https://github.com/akka/akka/issues/19822
Perhaps you would like to contribute to Akka also?